src/components/Settings/Notifications/NotificationsPushover/index.tsx

```diff
@@ -189,7 +190,8 @@ const NotificationsPushover: React.FC = () => {
                 </label>
                 <div className="form-input">
                   <div className="form-input-field">
-                    <Field
+                    <SensitiveInput
+                      as="field"
```

Comment:
```suggestion
                    <Field
```
As per our previous discussion, this field shouldn't be masked.

Reply:
The API token should not be protected?

Reply:
@ankarhem It's tied to the user account, it's not really an API token in the same sense as the others.  It is useless without the user token.  With Pushover, it is really easy to mix up the application and user tokens since they are quite similar (both 30-char alphanum strings), so I'm against masking them.